docs/_optimist/specs/images/index.de.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Images
+lang: de
+permalink: /optimist/specs/images/
+parent: Spezifikationen
+nav_order: 9500
+---
+
+# Images
+
+Es gibt 4 Arten von Images in OpenStack:
+
+- **Public Images:** Diese Images werden von uns gepflegt, sind für alle Benutzer verfügbar, werden regelmäßig aktualisiert und zur Verwendung empfohlen.
+- **Community Images:** Ehemals öffentliche Images, die durch neuere Versionen ersetzt wurden. Wir behalten diese Images, bis sie nicht mehr verwendet werden, um Ihre Deployments nicht zu gefährden.
+- **Private Images:** Von Ihnen hochgeladene Images, die nur für Ihr Projekt verfügbar sind.
+- **Shared Images:** Private Abbilder, die entweder durch Sie oder oder mit Ihnen in mehreren Projekten gemeinsam genutzt werden.
+
+Nur die ersten beiden Typen werden von uns verwaltet.
+
+## Public und community images
+
+Um Ihren Aufwand so gering wie möglich zu halten, stellen wir Ihnen eine Reihe ausgewählter Abbilder zur Verfügung.
+
+Aktuell enthält diese Liste:
+
+- Ubuntu 22.04 LTS (Jammy Jellyfish)
+- Ubuntu 20.04 LTS (Focal Fossa)
+- Ubuntu 18.04 LTS (Bionic Beaver)
+- Debian 11 (Bullseye)
+- Debian 10 (Buster)
+- CentOS 8
+- CentOS 7
+- Flatcar Linux
+- Windows Server 2019 (GUI/Core)
+
+Diese Images werden täglich auf neue Versionen überprüft. Die neueste verfügbare Version ist immer ein "public image" und endet auf `Latest`. Alle vorherigen Abbilder werden in "community images" umgewandelt, wir garantieren dabei nicht für eine endlose Verfügbarkeit. Wir werden sie so lange aufbewahren, wie sie aktiv genutzt werden. Community-Images haben das Datum des ersten Uploads in ihrem Namen.
```

Comment:
Community-Images haben das Datum des ersten Uploads in ihrem Namen. 
Community-Images werden durch unseren Automatismus umbenannt sobald eine aktuellere Version des jeweiligen Images verfügbar ist. Dabei wird "-Latest" mit dem initialen Upload Datum des Images ersetzt. zb.:...